src/pyhf/constraints.py

```diff
@@ -87,15 +87,10 @@ def _dataprojection(self, auxdata):
         normal_data = tensorlib.gather(auxdata, self.normal_data)
         return normal_data
 
-    def logpdf(self, auxdata, pars):
+    def make_pdf(self, pars):
         tensorlib, _ = get_backend()
         if not self.param_viewer.index_selection:
```

Comment:
when is param_viewer.index_selection None? I forgot.

Reply:
when on constrains of that type are defined/

Reply:
add a comment here. it's not obvious in this context.

Reply:
hm. not sure. The `if not self.param_viewer.index_selection` pattern is used throughout `constraints.py`. Not sure why it is less obvious here than elsewhere.